src/snmp_core.cc

```diff
@@ -18,6 +18,7 @@
 #include "comm/Connection.h"
 #include "comm/Loops.h"
 #include "fatal.h"
+#include "fde.h"
```

Comment:
Same question here: What src/snmp_core.cc scope does `fd_table was not declared in this scope` error message refer to?